pages/register.js

```diff
@@ -47,10 +60,87 @@ const RegistrationForm = () => {
                         justifyItems="start"
                         overflowY="auto"
                     >
-                        <FieldSection title='First Name' />
-                        <SystemComponent>as</SystemComponent>
-                        <SystemComponent>as</SystemComponent>
+                        <FieldSection 
+                        title='Full Name '
+                        asterik={true}                     
+                            name="fullName"
+                            value={formValues['fullName']}
+                            onChange={handleInputChange}        
+                            />
+                        <SystemComponent></SystemComponent>
+                    </SystemComponent>
+                    <SystemComponent
+                        display="grid"
+                        gridTemplateColumns="1fr"
+                        gridAutoRows="autofill"
+                        gridAutoFlow="row"
+                        gridGap={["cardPadding", "cardMargin", "cardMargin"]}
+                        justifyItems="start"
+                        overflowY="auto"
+                    >
+                        <FieldSection 
+                        title='Phone Number'      
+                        name="phoneNumber"
+                        value={formValues['phoneNumber']} 
+                        onChange={handleInputChange}
+                        />
+                        
+                        <SystemComponent></SystemComponent>
+                    </SystemComponent>
+                    <SystemComponent
+                        display="grid"
+                        gridTemplateColumns="1fr"
+                        gridAutoRows="autofill"
+                        gridAutoFlow="row"
+                        gridGap={["cardPadding", "cardMargin", "cardMargin"]}
+                        justifyItems="start"
+                        overflowY="auto"
+                    >
+                        <FieldSection 
+                        title='Personal Email Address'                             
+                        name="email"
+                        value={formValues['email']}
+                        onChange={handleInputChange}
+                        />
+
+                        <SystemComponent></SystemComponent>
                     </SystemComponent>
+                    <SystemComponent
+                        display="grid"
+                        gridTemplateColumns="1fr"
+                        gridAutoRows="autofill"
+                        gridAutoFlow="row"
+                        gridGap={["cardPadding", "cardMargin", "cardMargin"]}
+                        justifyItems="start"
+                        overflowY="auto"
+                    >
+                        <FieldSection 
+                        title='Program '
```

Comment:
Remove the space after 'Program '. If you want a space before the asterisk, you should be adding the space in the `TitleSection` React component you defined above.